crypto/x509/x509_vfy.c

```diff
@@ -204,9 +205,16 @@ static int check_auth_level(X509_STORE_CTX *ctx)
          * We've already checked the security of the leaf key, so here we only
          * check the security of issuer keys.
          */
-        if (i > 0 && !check_key_level(ctx, cert) &&
-            verify_cb_cert(ctx, cert, i, X509_V_ERR_CA_KEY_TOO_SMALL) == 0)
-            return 0;
+        if (i > 0) {
+            if (!check_curve(cert)
+                && verify_cb_cert(ctx, cert, i,
```

Comment:
Is the cb here so that the error can be ignored if needed?

Reply:
Yes, as in other similar places.